src/Sulu/Component/Webspace/Webspace.php

```diff
@@ -396,6 +396,30 @@ public function setNavigation($navigation)
     }
 
     /**
+     * Returns false if domain not exists in webspace.
+     *
+     * @param string $domain
+     * @param string $environment
+     *
+     * @return bool
+     *
+     * @throws Exception\EnvironmentNotFoundException
+     */
+    public function hasDomain($domain, $environment)
+    {
+        foreach ($this->getPortals() as $portal) {
+            foreach ($portal->getEnvironment($environment)->getUrls() as $url) {
+                $host = parse_url('http://' . $url->getUrl())['host'];
```

Comment:
Just tested `//` would also work, then we don't have any protocol information in here. This way somebody could think there is a reason for `http` here.